push-sdk-swift/DeviceRegistration.swift

```diff
@@ -232,7 +245,7 @@ open class DeviceRegistration: NSObject, URLSessionTaskDelegate {
         } else {
             value = Bundle.main.object(forInfoDictionaryKey: key) as? String
         }
-        if (value == nil && value!.isEmpty)  {
+        if (value == nil || value!.isEmpty)  {
```

Comment:
@jcesarmobile good catch this is the *really* wrong part. nil value and doing a forced  unwrapped. Where was my mind when I wrote that?